<launch>
   <include file="$(find motor_control_3dx)/launch/robot_start.launch" />
   <include file="$(find darknet_ros)/launch/yolo_v4.launch" />
   <include file="$(find azure_kinect_ros_driver)/launch/driver.launch" />


   <node name="people_tracking" pkg="people_tracking" type="people_tracking" output = "screen" >
     <param name="speed_Kp" value="0.15"/>
     <param name="speed_Kd" value="0.15"/>
     <param name="angle_Kp" value="0.002"/>
     <param name="angle_Kd" value="0.005"/>
     <param name="max_linear_speed" value="0.5"/>
     <param name="min_linear_speed" value="-0.2"/>
     <param name="max_angle_speed" value="1.0"/>
     <param name="target_distance" value="0.4"/>

   </node>

</launch>